src/Umbraco.Core/Persistence/Repositories/ContentRepository.cs

```diff
@@ -94,13 +94,7 @@ protected override IEnumerable<IContent> PerformGetByQuery(IQuery<IContent> quer
             return ProcessQuery(sql);
         }
 
-        #endregion
-
-        #region Static Queries
-
-        private readonly IQuery<IContent> _publishedQuery = Query<IContent>.Builder.Where(x => x.Published == true);

```

Comment:
Am I right in assuming this was never used anywhere? (doesn't look like it at least by looking at the dev-v7 branch)